sample_apps/simple_app_under_islet/islet_example_app.mak

```diff
@@ -42,12 +42,13 @@ O= $(OBJ_DIR)
 ISLET_S=$(S)/islet
 US=.
 I= $(SRC_DIR)/include
-INCLUDE= $(ISLET_INCLUDE) -I$(I) -I/usr/local/opt/openssl@1.1/include/ -I$(S)/sev-snp/ -I$(ISLET_S)
+INCLUDE= $(ISLET_INCLUDE) -I$(I) -I/usr/local/opt/openssl@1.1/include/ -I$(S)/sev-snp/ -I$(ISLET_S) -I.
+COMMON_SRC = ../common
```

Comment:
Could you move to specifying dirs off of CERTIFIER_ROOT; i.e. this becomes

COMMON_SRC = $(CERTIFIER_ROOT)/sample_apps/common